go/teams/list.go

```diff
@@ -244,7 +244,7 @@ func ListTeamsVerified(ctx context.Context, g *libkb.GlobalContext,
 		if team.IsImplicit() {
 			displayName, err := team.ImplicitTeamDisplayNameString(ctx)
 			if err != nil {
-				m.Debug("| Failed to get ImplicitTeamDisplayNameString() for team %q: %v", team.ID, err)
+				m.Warning("| Failed to get ImplicitTeamDisplayNameString() for team %q: %v", team.ID, err)
```

Comment:
sure this should be a warning?

Reply:
it's only used by the cli command if i'm not mistaken, and the result is empty displayName field in returned JSON, which was confusing when I was first debugging these issues. I can change to a `Debug` though, I don't feel strongly about this one.